content/10.diagnostics.md

```diff
@@ -173,4 +173,4 @@ Specifically, the individuals at risk include:
 
 However, this method of testing administration does not detect a high proportion of infections and does not allow for test-and-trace methods to be used.
 Individuals who are asymptomatic (i.e. potential spreaders) and individuals who are able to recover at home are therefore often unaware of their status.
-For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million against the 70,000 detected as of March 28th [@doi:10.25561/77731].
+For instance, a recent study from the Imperial College estimates that in Italy the true number of infections is around 5.9 million (total population ~60 million) against the 70,000 detected as of March 28th [@doi:10.25561/77731]; another example about New York state mentions for May 2020 ~300,000 cases reported (total population ~20 million) or ~1.5% of the population while antibody tests show ~20% positive [@url:ny.governor.gov/nyforwardbook].
```

Comment:
```suggestion
For instance, a recent study from Imperial College estimates that in Italy the true number of infections is around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population ~20 million, corresponding to ~1.5% of the population, but up to ~20% of the population tested positive when evaluated with antibody tests [@url:ny.governor.gov/nyforwardbook].
```
I'm not sure if this is what you were saying but I would recommend splitting up into two sentences just because it's so much info! I actually can't get the NYForwardBook url to work. In the [version I found](https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf) it sounds like NYC is ~20% but the rest of the state is lower?

Reply:
good catch! -- I had double-checked but I guess I should have triple checked! So, in addition to your suggested changes above we need to change the last number from ~20% to ~12%.
-so your changes work in how your rewrote the sentence
plus
-we need to change ~20% to ~12%
plus
-we should change the url to https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf
(I guessed this might be necessary)